packages/dht/src/dht/SortedContactList.ts

```diff
@@ -0,0 +1,132 @@
+import KBucket from 'k-bucket'
+import { PeerID } from '../helpers/PeerID'
+import { DhtPeer } from './DhtPeer'
+
+class ContactWrapper {
+    public contacted = false
+    public active = false
+    constructor(public contact: DhtPeer) {
+    }
+}
+
+export class SortedContactList {
```

Comment:
This is a priority queue with max size? Would it make sense to use e.g. `p-queue` or some other `npm` package?

Reply:
We want the SortedContactList to be a very custom solution. It is going to see a lot of changes in the trackerless-network PR including implementing EventEmitter.